compiler/rustc_mir/src/borrow_check/type_check/mod.rs

```diff
@@ -1456,9 +1456,7 @@ impl<'a, 'tcx> TypeChecker<'a, 'tcx> {
                 }
 
                 self.check_rvalue(body, rv, location);
-                if !(self.tcx().features().unsized_locals
-                    || self.tcx().features().unsized_fn_params)
-                {
+                if !self.tcx().features().is_unsized() {
```

Comment:
IMO it would be better to add a private helper in this file, the function seems awfully specific otherwise.

Also, "is" is a weird prefix here -- that sounds like it tests whether some particular type/variable *is* unsized. Maybe more like `unsized_feature()` or `unsized_feature_enabled()` or so?